src/styles/blocks/stars.scss

```
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+  align-items: center;
+  &__star {
```

please add empty line before inner rule
```suggestion
  align-items: center;
  
  &__star {
```

---

done! thank you!)